tv.sh

```diff
@@ -8,34 +8,38 @@ TITLES=(
   "SIC"
   "TVI"
   "RTP 3"
-  "SIC Noticias"
+  "SIC Notícias"
   "CNN Portugal"
-  "RTP Memoria"
+  "RTP Memória"
   "RTP Madeira"
-  "RTP Acores"
-  "RTP Africa"
+  "RTP Açores"
+  "RTP África"
   "RTP Internacional"
-  "SIC Internacional"
+  "TVI Internacional"
+  "TVI Ficção"
+  "TVI Reality"
   "ARTV"
   "Porto Canal"
   "Euronews"
   "Kuriakos TV"
 )
 
 STREAMS=(
-  "__rtp rtp1"
-  "__rtp rtp2"
-  "https://sic.live.impresa.pt/sic.m3u8"
+  "https://streaming-live-app.rtp.pt/liverepeater/smil:rtp1HD.smil/rtp_playlist.m3u8"
```

Comment:
This does not work on my machine, will try to debug why.

Reply:
RTP1 is again with DRM, yes :-(